src/shared/components/linkedIconCard/linkedIconCard.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styles from './linkedIconCard.css';
+
+// Stateless component that wraps a link around an icon / img and title
+// Image or Icon are passed as a child to allow more flexibility
+// Props = url, title, and at least one child
```

Comment:
I'd get rid of the comments here. It describes what's readily obvious and the prop-types are self-documenting.